components/brave_wallet/browser/filecoin_keyring.cc

```diff
@@ -80,7 +80,7 @@ std::string FilecoinKeyring::ImportFilecoinSECP256K1Account(
   auto payload = BlakeHash(uncompressed_public_key, 20);
   int protocol = static_cast<int>(mojom::FilecoinAddressProtocol::SECP256K1);
   std::string address = network + std::to_string(protocol) +
-                        GetAddressInternal(payload, protocol);
+                        CreateAddressWithProtocol(payload, protocol);
```

Comment:
you should just call `GetAddressInternal` here

Reply:
`CreateAddressWithProtocol` filecoin specific and  `GetAddressInternal` calls it, there is additional protocol parameter

Reply:
I meant these codes are duplicate
```
auto uncompressed_public_key = hd_key->GetUncompressedPublicKey();
  auto payload = BlakeHash(uncompressed_public_key, 20);
  int protocol = static_cast<int>(mojom::FilecoinAddressProtocol::SECP256K1);
  std::string address = network + std::to_string(protocol) +
                        CreateAddressWithProtocol(payload, protocol);
  ```
when you manage to get the network from settings, call `GetAddressInternal` to eliminate the duplicate

Reply:
there is a difference in network parameters, `GetAddressInternal` will create address using current network in wallet, but in import function that can be another imported address, the network is taken from parameters.